### DISCUSSION TITLE
Allow resizing of game logos.

### DISCUSSION THREAD

**Discussion by Cristobal15** [skyline-emu/skyline#799] [General]
It would be a good idea to allow the user to change the size of the game logos especially when using a tablet powerful enough to play Skyline. 

It is not a priority but it would be quite interesting in terms of aesthetics of the emulator.

**Comment by PixelyIon** [comment]
They're only 256x256 pixels and would look quite low resolution if they were to be larger than they already are.

**Reply by Cristobal15** [reply]
Well, you're right about that. Unfortunately the quality is very low.  At least to be able to resize the logos to something smaller, although thinking about it would also be a problem because of the space occupied by the name and it would not be aesthetic at all.
Thanks for the answer ✌️
